<launch>

<include file="$(find camera)/launch/camera.launch"/>

  <group ns="jetbot">

      <!-- remap the color_mask_node -->
      <node pkg="object_centroid" name="object_centroid_node" type="object_centroid.py">
          <remap from="in_topic" to="camera/processed/cropped"/>
          <remap from="out_topic" to="object_centroid"/>
      </node>

  </group>

</launch>
